packages/babel-plugin-transform-async-functions/src/index.js

```diff
@@ -1,5 +1,7 @@
+import { default as syntaxAsyncFunctions } from "babel-plugin-syntax-async-functions";
```

Comment:
Unfortunately that resulted in plugin options checker complaining something about invalid 'default' property.